### DISCUSSION TITLE
Standup (Wednesday, April 27, 2022)

### DISCUSSION THREAD

**Discussion by antonieto** [CarlosCRG19/ijalti-peers#25] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:
```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
 ```

**Comment by CarlosCRG19** [comment]
**What did you achieve in the last 24 hours?**:
 - Worked on #21 

**What are your priorities for the next 24 hours?**:
 - Finish #21 
 - Film mockup video
 - Probably define tech stack

**Blockers**:
 - Assisted to an event

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Not much 🤐

**What are your priorities for the next 24 hours?**:
 - Finish #17 

**Blockers**:
 - None

**Shoutouts**:
 - @antonieto for dropping by

**Comment by BrandonMagana** [comment]
**What did you achieve in the last 24 hours?**:
 -  Haven't done anything since finishig #23 

**What are your priorities for the next 24 hours?**:
 -  Record mock up video
 - Start defining tasks for next sprint

**Blockers**:
 - None

**Shoutouts**:
 - @DavBot02  @antonieto for Taco Tuesday

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - I finished half of sprint 0 mockup for sign up with @CarlosCRG19  #21 

**What are your priorities for the next 24 hours?**:
 - Finish that mockup asap
 - Define our tech stack

**Blockers**:
 - Nothing, puro pa' delante

**Shoutouts**:
 - @DavBot02 and @CarlosCRG19 for helping me moving to my new apartment

**Comment by CristianCazares** [comment]
**What did you achieve in the last 24 hours?**:
 - Introduction done and document done #12 , and aspirant search done #22 

**What are your priorities for the next 24 hours?**:
 - Help with what's left


**Shoutouts**:
 - @everyone for working nicely.
